selfdrive/controls/lib/events.py

```diff
@@ -183,6 +183,11 @@ def __init__(self, alert_text_1: str, alert_text_2: str, duration_text: float =
 
 
 # ********** alert callback functions **********
+def below_engage_speed_alert(CP: car.CarParams, sm: messaging.SubMaster, metric: bool) -> Alert:
+  speed = int(round(CP.minEnableSpeed * (CV.MS_TO_KPH if metric else CV.MS_TO_MPH)))
+  unit = "km/h" if metric else "mph"
```

Comment:
Can you abstract this into a `get_speed_unit(metric: bool)` helper function?

Reply:
I went ahead and extracted the whole speed conversion